EIPS/eip-3403.md

```diff
@@ -0,0 +1,69 @@
+* Refunds increase block size variance. The theoretical maximum amount of actual gas consumed in a block is nearly twice the on-paper gas limit (as refunds add gas space for subsequent transactions in a block, though refunds are capped at 50% of a transaction's gas used). This is [not fatal](https://notes.ethereum.org/@vbuterin/eip_1559_spikes), but is still undesirable, especially given that refunds can be used to maintain 2x usage spikes for far longer than EIP 1559 can.
+
+## Specification
+
+### Parameters
+
+| Constant | Value |
+| - | - |
+| `FORK_BLOCK` | TBD |
+
+For blocks where `block.number >= FORK_BLOCK`, the following changes apply.
+
+1. Remove the `SELFDESTRUCT` refund.
+2. Remove the `SSTORE` refund in all cases except for one specific case: if _new value_ and _original value_ both equal 0 but _current value_ does not, refund 15000 gas.
+
+## Rationale
+
+Preserving the one case `new = orig = 0 != current` ensures that a few key use cases that deserve favorable gas cost treatment continue to receive favorable gas cost treatment, particularly:
+
+* Anti-reentrancy locks (typically flipped from 0 to 1 right before a child call begins, and then flipped back to 0 when the child call ends)
+* ERC20 approve-and-send (the "approved value" goes from zero to nonzero when the token transfer is approved, and then back to zero when the token transfer processes)
+
+It also preserves two key goals of EIP 3298:
+
+1. Gas tokens continue to be non-viable, because each 15000 refund is only possible because of 15000 extra gas that was paid for flipping that storage slot from zero to nonzero earlier in the same transaction, so you can't clear some storage slots and use that saved gas to fill others.
+2. The total amount of gas _spent on execution_ is capped at the gas limit. Every 15000 refund for flipping a storage slot non from zero -> zero is only possible because of 15000 extra gas paid for flipping that slot from zero -> nonzero earlier in the same transaction; that gas paid for storage expansion and not execution, and so does not contribute to short-term DoS risk.
+
+## Backwards Compatibility
+
+Refunds are currently only applied _after_ transaction execution, so they cannot affect how much gas is available to any particular call frame during execution. Hence, removing them will not break the ability of any code to execute, though it will render some applications economically nonviable.
+
+[GasToken](https://gastoken.io/) in particular will become valueless. DeFi arbitrage bots, which today frequently use either established GasToken schemes or a custom alternative to reduce on-chain costs, would benefit from rewriting their code to remove calls to these no-longer-functional gas storage mechanisms.
```

Comment:
```suggestion
Gas tokens in particular will become valueless. DeFi arbitrage bots, which today frequently use either established gas token schemes or a custom alternative to reduce on-chain costs, would benefit from rewriting their code to remove calls to these no-longer-functional gas storage mechanisms.
```
There are multiple gas tokens, so changed this from a proper noun to a category of tokens.